server/tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/basic/api/SourceType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.graphql.tck.apps.basic.api;
+
+/**
+ * Just used to have e type usable with {@code @Source}
```

Comment:
```suggestion
 * Just used to have a type usable with {@code @Source}
```

Reply:
Sorry, this class shouldn’t be in this commit anyway!